code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -466,7 +466,9 @@
 
 // MELEE GANG
 /obj/item/claymore/ctf
-	slot_flags = SLOT_BACK
+	slot_flags = ITEM_SLOT_BACK
+	armour_penetration = 100
```

Comment:
Why tho? Their armor has 0 melee protection

Reply:
it's to pierce block